### PR TITLE
populate the pubspec repository field

### PR DESCRIPTION
**Pull request by devoncarew**
- populate the pubspec repository field

### CODE REVIEW

**Review by scheglov — APPROVED** [review]

**Comment by devoncarew** [comment]
@scheglov - I added a bit more to this PR (switching to using package:lints)

File: test/test_reflective_loader_test.dart [hunk 1]
```diff
@@ -2,6 +2,8 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
+// ignore_for_file: non_constant_identifier_names
```
Comment (scheglov) [inline test/test_reflective_loader_test.dart:5]: I cannot spot it, where is it reported?

**Review by scheglov — APPROVED** [review]
Still LGTM.

File: test/test_reflective_loader_test.dart (hunk 1 above)
Comment (devoncarew) [reply test/test_reflective_loader_test.dart:5]: I think there were the method names of the `TestReflectiveLoaderTest` class.

**Pull request merged**